src/Sql/Sql/Instance Failover Group/Cmdlet/RemoveAzureSqlInstanceFailoverGroup.cs

```diff
@@ -0,0 +1,172 @@
+        
+        /// <summary>
+        /// Parameter set name for remove with a resource ID.
+        /// </summary>
+        private const string RemoveIfgByResourceIdSet = "RemoveInstanceFailoverGroupByResourceId";
+
+        /// <summary>
+        /// Gets or sets the name of the resource group to use.
+        /// </summary>
+        [Parameter(ParameterSetName = RemoveIfgDefaultSet,
+            Mandatory = true,
+            Position = 0,
+            HelpMessage = "The name of the resource group.")]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public override string ResourceGroupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the local region to use.
+        /// </summary>
+        [Parameter(ParameterSetName = RemoveIfgByResourceIdSet,
+            Mandatory = true,
+            Position = 1,
+            HelpMessage = "The name of the Local Region from which to retrieve the Instance Failover Group.")]
+        [Parameter(ParameterSetName = RemoveIfgDefaultSet,
+            Mandatory = true,
+            Position = 1,
+            HelpMessage = "The name of the Local Region from which to retrieve the Instance Failover Group.")]
+        [LocationCompleter("Microsoft.Sql/locations/instanceFailoverGroups")]
+        [ValidateNotNullOrEmpty]
+        public override string Location { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the InstanceFailoverGroup to remove.
+        /// </summary>
+        [Parameter(ParameterSetName = RemoveIfgDefaultSet, 
+            Mandatory = true,
+            Position = 2,
+            HelpMessage = "The name of the Instance Failover Group to remove.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        /// <summary>
+	    /// Instance Failover Group object to remove
+	    /// </summary>
+	    [Parameter(ParameterSetName = RemoveIfgByInputObjectSet,
+            Mandatory = true,
+            Position = 0,
+            ValueFromPipeline = true,
+            HelpMessage = "The Instance Failover Group object to remove")]
+        [ValidateNotNullOrEmpty]
+        public AzureSqlInstanceFailoverGroupModel InputObject { get; set; }
+
+        /// <summary>
+        /// Gets or sets the resource ID of the Instance Failover Group to remove.
+        /// </summary>
+        [Parameter(ParameterSetName = RemoveIfgByResourceIdSet,
+            Mandatory = true,
+            Position = 0,
+            ValueFromPipeline = true,
+            HelpMessage = "The Resource ID of the Instance Failover Group to remove.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+        /// <summary>
+        /// Defines whether it is ok to skip the requesting of rule removal confirmation
+        /// </summary>
+        [Parameter(HelpMessage = "Skip confirmation message for performing the action.")]
+        public SwitchParameter Force { get; set; }
```

Comment:
@ziwa-msft do we need the `-Force` parameter here? Typically, the `-Force` parameter is only used in cases where we are prompting the user for additional confirmation. For example, if you are deleting a resource group that contains resources, you would want to prompt the user to confirm that they want to delete the resources inside the group in addition the group. If we're not deleting any additional resources outside of the failover group, than I don't think that we need the `-Force`.

Reply:
Hi Cormac. I believe the Force option should be preserved here because: 1) Dropping the failover group will also drop replication links for all databases within the group, which should meet your standard above. 2) it is a very sensitive operation as dropping the FG will leave all user dataabases within the group exposed to lack of DR proection. As a result we also have the Force option in the similar cmdlet used for Logical Server. 

Are you okay with keeping this option?

Reply:
@ziwa-msft sounds good to me